Formula/wireshark.rb

```diff
@@ -18,29 +18,32 @@ class Wireshark < Formula
   option "with-gtk+", "Build the wireshark command with gtk+"
   option "with-qt", "Build the wireshark command with Qt (can be used with or without either GTK option)"
   option "with-headers", "Install Wireshark library headers for plug-in development"
+  option "with-nghttp2", "Enable HTTP/2 header dissection"
 
+  depends_on "pkg-config" => :build
   depends_on "cmake" => :build
+  depends_on "c-ares"
+  depends_on "geoip"
   depends_on "glib"
   depends_on "gnutls"
   depends_on "libgcrypt"
-  depends_on "geoip"
-  depends_on "c-ares"
   depends_on "libsmi" => :optional
-  depends_on "lua"
+  depends_on "libssh" => :optional
+  depends_on "lua" => :optional
```

Comment:
Is it possible to mark this as recommended at least?

Reply:
maybe. will reply below.